go.mod

```diff
@@ -6,7 +6,6 @@ require (
 	github.com/aws-controllers-k8s/runtime v0.0.6
 	github.com/aws/aws-sdk-go v1.38.11
 	github.com/go-logr/logr v0.1.0
-	github.com/google/go-cmp v0.3.1
```

Comment:
hm ? upstream change ?

Reply:
Yeah, this is due to runtime v0.0.6 "delta" work replacing the use of the above library.

Reply:
+ `go mod tidy`